metecho/api/serializers.py

```diff
@@ -398,7 +398,7 @@ def get_valid_target_directories(self, obj) -> dict:
         return {}
 
     def validate(self, data):
-        if ScratchOrg.objects.filter(
+        if ScratchOrg.objects.active().filter(
```

Comment:
@wlonk I'm generally skeptical of this approach, where we need to remember to use `active()` all over the place. I'd be tempted to make that the default, such that we have to explicitly get deleted objects when we want them.

Reply:
I've run into problems with that in the past, so either way is a bit hazardous. I agree, in principle, though.